test/cppsuite/tests/hs_cleanup.cpp

```diff
@@ -111,7 +103,6 @@ class hs_cleanup : public test {
             testutil_assert(rollback_retries < MAX_ROLLBACKS);
         }
         /* Ensure our last transaction is resolved. */
-        if (tc->txn.active())
-            tc->txn.rollback();
+        tc->txn.try_rollback();
```

Comment:
I suspect there are a number of other tests that can have the:
```
if (tc->txn.active())
    tc->txn.rollback();
```
updated.

Reply:
Done in d809bcd6da19d31def2810b2e10fab2cae16d4db